policy/release/trusted_task_test.rego

```diff
@@ -435,6 +456,22 @@ task_c := {
 	"ref": {"name": "TaskC", "kind": "Task", "bundle": trusted_bundle},
 }
 
+task_test_a := {
+	"metadata": {"labels": {"tekton.dev/pipelineTask": "task_a"}},
+	"invocation": {"parameters": {"B_ARTIFACT": artifact_b, "D_ARTIFACT": artifact_d}},
+	"results": [{
+		"name": "TEST_OUTPUT",
+		"value": "{\"FAILED\": \"1\"}",
+		"type": "string",
+	}],
+	"ref": {"name": "TaskA", "kind": "Task", "bundle": trusted_bundle},
+}
+
+attestation_test_ta := {"statement": {"predicate": {
```

Comment:
We could use the existing `attestation_ta` statement and just add `task_test_a` to it? (You'd have to change the value of `tekton.dev/pipelineTask` to be unique (and also probably the `.ref.name` value)

Reply:
Good idea. That better reflects the attestations we work with.